srcpkgs/wyrd/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'wyrd'
+pkgname=wyrd
+version=1.5.3
+revision=1
+build_style=gnu-configure
+#configure_args=""
```

Comment:
Remove these lines

Reply:
I'm confused, aren't those lines necessary? Did you mean lines 6-8?